src/App.js

```diff
@@ -44,7 +51,8 @@ class App extends Component {
 
     this.setState({
       nextSelectedValue,
-      selectedValues: newSelectedValues
+      selectedValues: newSelectedValues,
+      lastFiveSelectedValues: lastFiveValues
```

Comment:
Nit: If the local variable here were called `lastFiveSelectedValues` instead of `lastFiveValues`, we could use just 
```
  lastFiveSelectedValues,
``` 
here instead